UpdateOrInstall.lua

```diff
@@ -356,7 +356,9 @@ function UpdateOrInstall.buildCopyFilesCommand(extractedFolder, isOnWindows)
 			string.format('echo %s', messages.filesready),
 			string.format('cd "%s"', IronmonTracker.workingDir), -- required for mGBA on Windows
 			string.format('echo %s', messages.updating),
-			string.format('xcopy "%s" /s /y /q', extractedFolder),
+			-- /s: for subdirectories, /y: no overwrite prompts, /q: no msg display, /c: skip files with errors
+			string.format('xcopy "%s" /s /y /q /c', extractedFolder),
```

Comment:
mentioned in the discord but also putting here for completeness:

Main concern with this is what if the file that would cause an error and therefore be skipped by the `/c` flag is a file we *want* to update? Skipping would then cause a partial update to happen which could just break things even more depending on what the skipped files are, so how do we know for sure that the skipped files are *only* ones we don't need to update?